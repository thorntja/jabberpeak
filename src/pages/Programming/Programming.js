import React from 'react';

const Programming = () => (
  <div>
    This is the programming section!
  </div>
)

export default Programming;
