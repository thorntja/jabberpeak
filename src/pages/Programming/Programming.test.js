import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Programming from './Programming';

test('Programming page should render as expected', () => {
  const component = shallow(<Programming />);
  const tree = toJson(component)
  expect(tree).toMatchSnapshot();
})
