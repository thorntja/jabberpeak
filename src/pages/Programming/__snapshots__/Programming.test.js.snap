// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Programming component should render as expected 1`] = `
<div>
  This is the programming section!
</div>
`;
